compiler/AST/primitive.cpp

```diff
@@ -96,18 +101,12 @@ returnInfoDefaultInt(CallExpr* call) {
   return returnInfoInt64(call);
 }
 
-static Type*
-returnInfoUInt64(CallExpr* call) {
-  return dtUInt[INT_SIZE_64];
-}
-
```

Comment:
Works for me.  I must admit that I didn't look at the larger context for this one -- it just caught my eye given the other.

I got to the runtime (taking only the highest-level read-through of the types of changes), but am heading out for the night.  Looks nice so far!